opencensus/proto/agent/trace/v1/BUILD.bazel

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+load("@grpc_java//:java_grpc_library.bzl", "java_grpc_library")
+
+proto_library(
```

Comment:
I think we should also have the go generated using bazel rules:
https://github.com/bazelbuild/rules_go/blob/master/proto/core.rst#example-grpc

Reply:
I think we're directly depending on the gen-go files in Go? @rakyll 

Adding `go_proto_library` build rule will introduce a lot of transitive dependencies.

Reply:
I left a TODO here for adding the `go_proto_library` rule if needed. That will be a lot of code changes so I think it's better to do it in a separate PR.